### ISSUE TITLE
Not showing receive message channel

### ISSUE THREAD

**Issue by arisliang** [Dorin-Mocan/SignalRSwaggerGen#34]
The swagger shows endpoints to send message to nicely for strongly typed hub. But it doesn't show the receive endpoints, which is defined in the interface T of Hub<T> for a strongly typed hub.

E.g. strongly typed hub is defined created as

```
public interface IChatClient
{
    // receive channel                        - no, this doesn't show.
    Task ReceiveMessage(string user, string message);
}

[SignalRHub]
public class StronglyTypedChatHub : Hub<IChatClient>
{
    // send channel                           - yes, shown in swagger.
    public async Task SendMessage(string user, string message)
    {
        await Clients.All.ReceiveMessage(user, message);
    }
}
```

Anyway we could show the receive channel as well?

**Comment by Dorin-Mocan** [comment]
@arisliang hi! You need to decorate the interface with [SignalRHub] as well. Only interfaces/classes/structs decorated with [SignalRHub] are scanned. In order to support my work, please toss a start for this repo. If there is anything else I can help with, let me know. Thank you!

**Comment by arisliang** [comment]
After adding `[SignalRHub] to the interface, it shows up in the doc. But the routes are not showing according to the MapHub endpoints, and it's not configurable by default `Route` attribute, which only work on class, not interface.

It would be more useful to have a custom `SignalRRoute` attribute, to specify the display of the hub interface's routes.

**Comment by Dorin-Mocan** [comment]
@arisliang [SignalRHub] has a parameter for specifying the path/route. Check out.

**Comment by arisliang** [comment]
`[SignalRHub(path:)]` worked for route, great!

Is there a summary option for documenting the hub? It doesn't seem to pick up from summary as usually swagger does. Also `[SignalRMethod]` has a summary option for swagger display, can SignalRHub specify summary as well?

**Comment by arisliang** [comment]
As this issue is closed, I open another one #35 for the summary display.
